src/core_int/integration_slack.py

```diff
@@ -18,10 +18,29 @@
 LOG = logging.getLogger('integration')
 
 
-def integration_slack_receive_dm_main(
+@dataclass
+class IntegrationSlackSendMessageEnv(IntegrationEnvironment):
+    slackApiToken: str
+
+
+def integration_slack_main(
         env: 'IntegrationEnvironment',
         events: 'IntegrationEvents'):
 
+    sc = WebClient(env.slackApiToken, run_async=True)
+
+    @events.ledger.contract_created(
+        'SlackIntegration.OutboundMessage.OutboundMessage')
+    async def on_contract_created(event):
+        LOG.info('slack send message - created: %r', event)
+
+        channel = event.cdata['slackChannel']
+        text = event.cdata['messageText']
+
+        sc.chat_postMessage(channel=channel, text=text)
+
+        return [exercise(event.cid, 'Archive')]
+
     @events.webhook.post(label='Slack Event Webhook Endpoint')
     async def on_webhook_post(request):
         body = await request.json()
```

Comment:
`500` status if this fails, due to response being None:
https://github.com/digital-asset/daml-dit-if/blob/master/daml_dit_if/main/integration_webhook_context.py#L48

Reply:
500 status if parsing the request fails?

Reply:
yes.
```bash
curl -X POST https://integrations.projectdabl.com/integration/ijo8doscmuqkyicz
500 Internal Server Error
```

Reply:
I think that's probably acceptable, although we do need to do some overall robustness work on this integration as part of a later PR.

Reply:
yep my thoughts as well.